zerver/lib/scheduled_messages.py

```diff
@@ -11,3 +13,13 @@ def access_scheduled_message(
         return ScheduledMessage.objects.get(id=scheduled_message_id, sender=user_profile)
     except ScheduledMessage.DoesNotExist:
         raise ResourceNotFoundError(_("Scheduled message does not exist"))
+
+
+def get_all_scheduled_messages(user_profile: UserProfile) -> List[ScheduledMessageDict]:
```

Comment:
`get_undelivered_scheduled_messages` could be better, but I'm not sure the distinction is important.